stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -132,11 +132,18 @@ public static InspectorModulesProvider defaultInspectorModulesProvider(final Con
       @Override
       public Iterable<ChromeDevtoolsDomain> get() {
         ArrayList<ChromeDevtoolsDomain> modules = new ArrayList<ChromeDevtoolsDomain>();
-        modules.add(new Console());
         modules.add(new CSS());
         modules.add(new Debugger());
+        Runtime runtime = new Runtime();
+        modules.add(runtime);
         if (Build.VERSION.SDK_INT >= AndroidDOMConstants.MIN_API_LEVEL) {
-          modules.add(new DOM(new AndroidDOMProviderFactory((Application)context.getApplicationContext())));
+          DOM dom = new DOM(
+              new AndroidDOMProviderFactory(
+                  (Application)context.getApplicationContext()));
+          modules.add(dom);
+          modules.add(new Console(dom, runtime));
```

Comment:
we should probably talk with @ichub about this. In his refactoring PR #237 we are splitting DOM into two classes, DOM and Document, with the intent that CSS will have a reference to Document in order to avoid having a reference to DOM. In this case you're giving Console access to DOM and Runtime and that may not be the right approach, especially since you don't actually need DOM -- you need the ObjectIdMapper.